apps/src/studio/Item.js

```diff
@@ -18,12 +18,15 @@ var Item = function (options) {
   this.height = options.height || 50;
   this.width = options.width || 50;
   this.speed = options.speed || constants.DEFAULT_SPRITE_SPEED / 2;
+  this.animationFrames = options.animationFrames || 1;
 
   this.currentFrame_ = 0;
   var self = this;
   this.animator_ = window.setInterval(function () {
-    if (self.loop || self.currentFrame_ + 1 < self.frames) {
-      self.currentFrame_ = (self.currentFrame_ + 1) % self.frames;
+    if (self.dir != Direction.NONE) {
+      if (self.loop || self.currentFrame_ + 1 < self.frames) {
+        self.currentFrame_ = (self.currentFrame_ + 1) % self.frames;
+      }
```

Comment:
Could combine if statement into a single line. Alternatively, could invert and reducing nesting, i.e...
```
if (self.dir === Direction.NONE) {
  return;
}
if (self.loop...) {
  self.currentFrame_ = ...
}
```

Also, instead of using self, we could just bind the function inside of setInterval to this, i.e.

```
window.setInterval(function () {
  if (this.dir != Direciton.NONE) {
  ...
}.bind(this), 50)